classes/item.rb

```diff
@@ -1,14 +1,21 @@
 require 'date'
+
 class Item
   attr_reader :id, :archived, :label
```

Comment:
You have done a great job while implementing the has-many/belongs-to relationship between the Genre class and the Item class. However, one thing I would recommend adding here in the Item class is the reading permission for the genre variable because later on, someone may need to read the genre for a particular item object and without that permission, it is not possible. 
I kindly recommend adding it there too. Like this
![Review](https://user-images.githubusercontent.com/90222110/165129123-68ba7154-555e-4a91-adab-bb43ab624e79.png)
😃  💡  👍

Reply:
changes resolved